File: 44_lab1.cpp

#include<iostream>
using namespace std;
int main()
{
	int n1,n2,temp;
	cout << "enter first number" << endl;
	cin >> n1;
	cout << "enter second number" <<endl;
	cin >> n2;
	cout <<"First Number="<<n1<<endl<<"Second Number="<<n2<<endl;
	cout << "Addition of " <<n1<<" and "<<n2<<" is "<<n1+n2;
	cout<<"\nSwapping \n";
	temp=n1;
	n1=n2;
	n2=temp;
	cout<<"Swapped numbers n1= "<<n1<<" and n2= "<<n2<<endl;
	cout<<"even or odd \n";
	if(n1%2==0)
		cout<<n1<<" is even \n";
	else
		cout<<n1<<" is odd \n";
	
	return 0;
}

File: 44_lab1.java
import java.util.Scanner;

public class Lab1 {

	public static void main(String[] args) {

	    Scanner sc = new Scanner(System.in);  // Create a Scanner object

	   /* System.out.println("Enter username");



	    String userName = sc.nextLine();  // Read user input

	    System.out.println("Username is: " + userName);  // Output user input

	 */

	    int n1,n2,temp;

	    System.out.println("Enter first number");

	    n1=sc.nextInt();

	    System.out.println("Enter second number");

	    n2=sc.nextInt();

	    

	    System.out.println("Number 1 = "+n1+" Number 2 = "+n2);

	    System.out.println("\n ADDITION\n");

	    System.out.println("\nAddition of both numbers is: " +(n1+n2));

	    

	    System.out.println("\n SWAPPING\n");

	    temp=n1;

	    n1=n2;

	    n2=temp;

	    System.out.println("After swapping Number 1 = "+n1+" Number 2 = "+n2);

	    

	    System.out.println("\n EVEN/ODD\n");

	    if(n1%2==0)

	    	System.out.println(n1+" is Even");

	    else

	    	System.out.println(n1+" is Odd");

  

	  }

}

